### PR TITLE
Tag name update

### PR DESCRIPTION
**Pull request by zolopane**
Removing white space. I did a million commits to get this one in, but I think this is correctly updated to main except for the white space

### CODE REVIEW

File: lob-api-public.yml [hunk 1]
```diff
@@ -1108,7 +1108,7 @@ tags:
         }
       ```
 
-  - name: Requests and Responses
+  - name: RequestsandResponses
```
Comment (sudoku-lord) [inline lob-api-public.yml:1111]: Similar to the previous nitpick: can we change this to `RequestsAndResponses`?

File: lob-api-public.yml [hunk 2]
```diff
@@ -1608,15 +1608,15 @@ tags:
...
+  - name: TemplateVersions
     description: |
       These API endpoints allow you to create, retrieve, update and delete versions of reusable HTML templates for use with the Print & Mail API.
 
   - name: Templates
     description: |
       These API endpoints allow you to create, retrieve, update and delete reusable HTML templates for use with the Print & Mail API.
 
-  - name: Test and Live Environments
+  - name: TestandLiveEnvironments
```
Comment (sudoku-lord) [inline lob-api-public.yml:1619]: Can we use `TestAndLiveEnvironments`?

File: lob-api-public.yml [hunk 3]
```diff
@@ -2113,7 +2113,7 @@ tags:
       The test API key does not perform any verification, automatic correction, or standardization for addresses. If you wish to
       try these features out, use our [live demo](https://lob.com/address-verification) or the free plan (see [our pricing](https://lob.com/pricing/address-verification) for details).
 
-  - name: Versioning and Changelog
+  - name: VersioningandChangelog
```
Comment (sudoku-lord) [inline lob-api-public.yml:2116]: The same nitpick: `VersioningAndChangelog`

File: resources/reverse_geocode_lookups/reverse_geocode_lookups.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@ post:
     Reverse geocode a valid US location with a live API key.
 
   tags:
-    - Reverse Geocode Lookups
+    - ReverseGeocode
```
Comment (sudoku-lord) [inline resources/reverse_geocode_lookups/reverse_geocode_lookups.yml:10]: This should be `ReverseGeocodeLookups` as per `lob-api-public.yml`.

File: resources/zip_lookups/zip_lookups.yml [hunk 5]
```diff
@@ -7,7 +7,7 @@ post:
     Returns information about a ZIP code
 
   tags:
-    - Zip Lookups
+    - Zip
```
Comment (sudoku-lord) [inline resources/zip_lookups/zip_lookups.yml:10]: This should be `ZipLookups` as per `lob-api-public.yml`.

File: lob-api-public.yml [hunk 6]
```diff
@@ -2278,55 +2278,55 @@ x-tagGroups:
...
+      - BetaProgram
       - Errors
-      - Rate Limiting
-      - Requests and Responses
-      - Test and Live Environments
-      - Versioning and Changelog
+      - RateLimiting
+      - RequestsandResponses
+      - TestandLiveEnvironments
+      - VersioningandChangelog
```
[the remark this answers is not in the document]
Comment (SidneyAllen) [reply lob-api-public.yml]: @zolopane - did you see all thse requested changes?

**Review by sudoku-lord — APPROVED** [review]

**Pull request merged**